src/components/Text/RegularText/RegularTextStyled.ts

```diff
@@ -0,0 +1,15 @@
+import styled from 'styled-components';
+import { BaseText } from '../../../style/baseText';
+import { theme } from '../../../style/theme';
+
+interface TextProps {
+  isBaseColor: boolean;
+  isBold: boolean;
+}
+
+export const RegularTextStyled = styled(BaseText)`
+  color: ${(props: TextProps) =>
+    props.isBaseColor ? theme.black : theme.white};

```

Comment:
Rather than a `isBaseColor` prop, maybe create a primary and secondary variant to handle the text color change

Reply:
Right, ok will fix it like that